### DISCUSSION TITLE
can not extract

### DISCUSSION THREAD

**Discussion by cykky** [MRCIEU/TwoSampleMR#496] [Q&A]
how to solve this problem:
> x<-extract_instruments(outcomes = "ieu-b-40")
Error in api_query("tophits", query = list(id = id, pval = pval, preclumped = preclumped,  : 
  The query to MR-Base exceeded 300 seconds and timed out. Please simplify the query

**Comment by YangXiaorong1987** [comment]
Hi I met the same problems like you. did you solve this?
